src/lib/flows/claim-project-flow/steps/poll-api/poll-api.svelte

```diff
@@ -46,6 +47,38 @@
       }
     `;
 
+    assert($context.gaslessOwnerUpdateTaskId, 'Gasless owner update task ID is missing');
```

Comment:
Does assert eventually produce some sort of global error display for the user?

Reply:
Yep, any error thrown within the context of a `Stepper` will be caught by the stepper component and trigger an `Error` step that just stringifies the error and shows it to the user.